reference/6/PowerShellGet/Find-DscResource.md

```diff
@@ -122,8 +147,10 @@ Accept wildcard characters: False
 ```
 
 ### -AllVersions
+
 Specifies that you want to include all available versions of a module in results.
-You cannot use the *AllVersions* parameter with the *MinimumVersion*, *MaximumVersion*, or *RequiredVersion* parameters.
+You cannot use the **AllVersions** parameter with the *MinimumVersion*, *MaximumVersion*, or
```

Comment:
MinimumVersion and MaximumVersion missing bold.

Reply:
fixed